packages/geo-diary-v2/src/core/router.tsx

```diff
@@ -7,8 +7,30 @@ import PrivateRouteWrapper from './private-route-wrapper'
 
 export const history = createBrowserHistory()
 
-const LoginPage = React.lazy(() => import('../components/pages/login'))
-const AuthenticatedPage = React.lazy(() => import('../components/pages/authenticated'))
+export const catchError = (
```

Comment:
I think this function and its tests should be moved to elements/utils, then we can import and use it everywhere

Reply:
@vuhuucuong I'll move to common package, and move sentry logger to common package instead of let it outside